### PR TITLE
{devel}[intel/2015b] PyZMQ v15.1.0, ZeroMQ v4.1.4, util-linux v2.27.1, ... (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-intel-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ that enables receivers to detect loss, request retransmission of lost data, or notify an application
+ of unrecoverable loss. PGM is a receiver-reliable protocol, which means the receiver is responsible
+ for ensuring all data is received, absolving the sender of reception responsibility."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [GOOGLECODE_SOURCE]
+sources = ['libpgm-%(version)s.tar.gz']
+
+configopts = '--with-pic'
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-intel-2015b.eb]: `pic: True`?

File: easybuild/easyconfigs/u/util-linux/util-linux-2.27.1-intel-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+source_urls = ['%s/v%%(version_major_minor)s' % homepage]
+sources = [SOURCELOWER_TAR_GZ]
+
+# disable account related utilities (they need OS dependant pam-devel files)
+# disable wall and friends (requires group changing permissions for install user)
+# install systemd service files in install dir
+# install bash completion files in install dir
+configopts = '--disable-chfn-chsh --disable-login --disable-su '
+configopts += '--disable-wall --disable-use-tty-group '
+configopts += '--disable-makeinstall-chown --disable-makeinstall-setuid '
```
Comment (wpoely86) [inline easybuild/easyconfigs/u/util-linux/util-linux-2.27.1-intel-2015b.eb]: use double quotes as below?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e15ea3b43ddbeea98c55 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**